_posts/2016-12-22-Rust-1.14.md

```diff
@@ -0,0 +1,356 @@
+---
+layout: post
+title: "Announcing Rust 1.14.0"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce the latest version of Rust, 1.14.0. Rust is a
+systems programming language focused on safety, speed, and concurrency.
+
+As always, you can [install Rust 1.14.0][install] from the appropriate page on our
+website, and check out the [detailed release notes for 1.14.0][notes] on GitHub.
+1230 patches were landed in this release.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1140-2016-12-22
+
+### What's in 1.14.0 stable
+
+One of the biggest features in Rust 1.14 isn't actually in Rust 1.14: the
```

Comment:
Possible change: "One of the biggest features in Rust 1.14 isn't actually in the language itself:"?